ui/shared/AddressInput.qml

```diff
@@ -86,6 +112,33 @@ Item {
             }
         }
     }
+
+    Loader {
+        sourceComponent: loadingIndicator
+        anchors.top: inpAddress.bottom
+        anchors.right: inpAddress.right
+        anchors.topMargin: 8
+        active: root.isPending
+    }
+
+    Component {
+        id: loadingIndicator
+        SVGImage {
```

Comment:
We already have a Loading image component: https://github.com/status-im/nim-status-client/blob/master/ui/shared/LoadingImage.qml